### PR TITLE
mail collectors: added debug information #1078

### PR DESCRIPTION
**Pull request by e3rd**
It seems to me these log lines will suit to most test cases.

### CODE REVIEW

**Comment by e3rd** [comment]
Thanks for the advices! Any other thing I can do here?

> dot

I did an regex search and added puctuation to multiple places in the project.

> parameters for string formatting are better passed as argument to the log function

What does that mean please? I'm already passing the parameters as arguments. See, subject is passed as an argument. `self.logger.debug("Message %s skipped.", message.subject)` 

> Another possible debug output:

I did rather "no unred mails to check" because there may be other parameters than read/unread (also sent-to/from).

File: intelmq/bots/collectors/mail/collector_mail_attach.py [hunk 1]
```diff
@@ -37,6 +38,7 @@ def process(self):
                 if (self.parameters.subject_regex and
                         not re.search(self.parameters.subject_regex,
                                       re.sub("\r\n\s", " ", message.subject))):
+                    self.logger.debug("Message %s skipped", message.subject)
```
[the remark this answers is not in the document]
Comment (e3rd) [reply intelmq/bots/collectors/mail/collector_mail_attach.py]: implemented

File: intelmq/bots/collectors/mail/collector_mail_url.py [hunk 2]
```diff
@@ -46,9 +47,10 @@ def process(self):
                 if (self.parameters.subject_regex and
                         not re.search(self.parameters.subject_regex,
                                       re.sub("\r\n\s", " ", message.subject))):
+                    self.logger.debug("Message with date %s skipped because subject %r does not match.", message.subject)
```
Comment (wagner-certat) [inline intelmq/bots/collectors/mail/collector_mail_url.py]: There's a parameter missing

**Comment by wagner-certat** [comment]
> > parameters for string formatting are better passed as argument to the log function

> What does that mean please?

-.- Sorry, don't know where I looked at.

> I did an regex search and added puctuation to multiple places in the project.

Thanks!

> I did rather "no unred mails to check" because there may be other parameters than read/unread (also sent-to/from).

Apart from the one missing parameter, ready to merge. I think it's sufficient for most purposes.

File: intelmq/bots/collectors/mail/collector_mail_url.py (hunk 2 above)
Comment (e3rd) [reply intelmq/bots/collectors/mail/collector_mail_url.py]: added here and in the other file, too

**Comment by e3rd** [comment]
sorry, my bad – added ✓

**Comment by wagner-certat** [comment]
merged and fixed the code style too (travis/pycodestyle complained about 2 too long lines)

**Pull request merged**